pandas/tests/io/parser/common/test_common_basic.py

```diff
@@ -86,7 +86,9 @@ def test_read_csv_local(all_parsers, csv1):
 
     fname = prefix + str(os.path.abspath(csv1))
     result = parser.read_csv(fname, index_col=0, parse_dates=True)
-
+    # TODO: make unit check more specific
+    if parser.engine == "pyarrow":
```

Comment:
Shouldn't we have a pyarrow version compat instead of blanket doing this? I think we should just take whatever precision pyarrow provides?